StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -2095,13 +1990,16 @@ public class STPPaymentHandler: NSObject {
     }
 
     // MARK: - Errors
-    /// - Parameter loggingSafeUserInfo: Error details that are safe to log i.e. don't contain PII/PDE or secrets.
+    /// - Parameter loggingSafeErrorMessage: Error details that are safe to log i.e. don't contain PII/PDE or secrets.
     @_spi(STP) public func _error(
         for errorCode: STPPaymentHandlerErrorCode,
         apiErrorCode: String? = nil,
-        loggingSafeUserInfo: [String: String] = [:]
+        loggingSafeErrorMessage: String? = nil,
+        localizedDescription: String? = nil
     ) -> NSError {
-        var userInfo = loggingSafeUserInfo
+        var userInfo = [String: String]()
+        userInfo[STPError.errorMessageKey] = loggingSafeErrorMessage
```

Comment:
Sorry, i didn't catch this before.. before we were calling:
```
_error( ...,
   loggingSafeUserInfo: ["error_message": "..."]
)
```

But now we end up calling:
```
_error(...,
      loggingSafeErrorMessage: "..."
)
//and then:
["com.stripe.lib:ErrorMessageKey": loggingSafeErrorMessage]
```
I'm not quite sure what key is expected here, but something seems off?

Reply:
Talked offline and resolved